### DISCUSSION TITLE
Node-Selection unavailable during Wallet Setup

### DISCUSSION THREAD

**Discussion by MaKla89** [iotaledger/firefly#1542] [Features / Enhancements]
## Issue description

The option to change the node configuration seems to be unavailable during the initial wallet setup:
![image](https://user-images.githubusercontent.com/33317175/111900627-f26b1000-8a33-11eb-8472-bf21ca6ed2ea.png)

In situations where the default node setup is not working this leads to a dead end where you cannot finish wallet setup due to a bad node configuration and you need to finish the wallet setup in order to change the node configuration:

![image](https://user-images.githubusercontent.com/33317175/111900661-27776280-8a34-11eb-87fc-f3041893ffba.png)

## Wallet version

0.2.1

## System specification

Windows 10 (Build 19042.867)

## Is it a repeatable issue?

Not tested

## Steps To reproduce the issue

Explain how the maintainer can reproduce the issue.

1. Create new account / wallet while default nodes are not available
2. Get to the point "Back up to a file"
3. Independtly from your choice ("Skip" or "Save backup") an error should occur
4. Try changing the node configuration, it should be disabled

## Expected behaviour

Me being allowed to change node configuration at any given time, even during the initial setup

## Actual behaviour

I'm not able to access the node configuration during the initial firefly setup, leading to me getting stuck.

**Comment by maxwellmattryan** [comment]
Soon we'll have developer profiles released with Firefly, and I'm thinking that this would be perfect to add for if the user selects to create a developer profile. At the moment we have it defaulting to the devnet, but it's likely that a user would still be creating the profile to add their own node so we should make sure they can do that more quickly (perhaps to check if it's healthy or something).
